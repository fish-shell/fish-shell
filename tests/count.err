
####################
# no args

####################
# one args

####################
# two args

####################
# args that look like flags or are otherwise special

####################
# big counts

####################
# stdin
