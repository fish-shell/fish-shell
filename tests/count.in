# Validate the behavior of the `count` command.

logmsg no args
count

logmsg one args
count x

logmsg two args
count x y

logmsg args that look like flags or are otherwise special
count -h
count --help
count --
count -- abc
count def -- abc

logmsg big counts

# See #5611
for i in seq 500
    set c1 (count (seq 1 10000))
    test $c1 -eq 10000
    or begin
        echo "Count $c1 is not 10000"
        break
    end
end
