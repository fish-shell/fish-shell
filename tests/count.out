
####################
# no args
0

####################
# one args
1

####################
# two args
2

####################
# args that look like flags or are otherwise special
1
1
1
2
3

####################
# big counts
